packages/neon/neon/lib/utils.dart

```diff
@@ -0,0 +1,12 @@
+export 'package:neon/src/utils/app_implementation.dart';
```

Comment:
I never liked this being a util.

It should rather be a model or interface but I don't know how much it makes sense :(